timeseries/src/autogluon/timeseries/metrics/point.py

```diff
@@ -392,7 +392,9 @@ def __init__(self, alpha: float = 0.5):
             f"{self.name} is an experimental metric. Its behavior may change in the future version of AutoGluon."
         )
 
-    def save_past_metrics(self, data_past: TimeSeriesDataFrame, **kwargs) -> None:
+    def save_past_metrics(
+        self, data_past: TimeSeriesDataFrame, target: str = "target", seasonal_period: int = 1, **kwargs
```

Comment:
`seasonal_period` is unused?

Reply:
The abstract method defines it. Not repeating the definition here violates the Liskov substitution principle, since

```
scorer: TimeSeriesScorer = WCD()

scorer.save_past_metrics(1, 2)  # TypeError
```